### PR TITLE
Add cxxopts package

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

**Comment by Alexpux** [comment]
@bwrsandman add header to PKGBUILD with maintainer info

**Comment by bwrsandman** [comment]
Myself, you mean?

**Comment by Alexpux** [comment]
Yes

**Comment by bwrsandman** [comment]
It's done

File: mingw-w64-cxxopts/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+pkgrel=1
+pkgdesc="Lightweight C++ command line option parser (mingw-w64)"
+arch=('any')
+url="https://github.com/jarro2783/cxxopts"
+license=("MIT")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-gcc"
+             "git")
+options=('strip' 'staticlibs')
+source=(${_realname}::https://github.com/jarro2783/cxxopts/archive/v${pkgver}.tar.gz)
```
Comment (Alexpux) [inline mingw-w64-cxxopts/PKGBUILD:15]: it must be ````${_realname}-${pkgver}.tar.gz````

File: mingw-w64-cxxopts/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    ../cxxopts-${pkgver}
+
+  make
+}
+
+package () {
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  make DESTDIR="${pkgdir}" install
+
+  install -D -m644 ${srcdir}/cxxopts-${pkgver}/LICENSE "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
```
Comment (Alexpux) [inline mingw-w64-cxxopts/PKGBUILD:39]: MIngw packages never install in USR

**Comment by bwrsandman** [comment]
This error doesn't seem related to this changeset:
```
==> ERROR: Cannot find the /usr/bin/zstd binary required for makepkg.
```

**Pull request merged**